src/components/Timer/getTime.ts

```diff
@@ -1,13 +1,14 @@
 import { TrackPath, Challenge } from '../../model/ChallengeConfiguration'
 import * as turf from '@turf/turf'
 import { DateTime as LuxonDate } from 'luxon'
+import { Duration } from 'luxon'
 
 interface Props {
   path: TrackPath
   challenge: Challenge
 }
 
-const getTimes = ({ path, challenge }: Props): number | null => {
+const getTimes = ({ path, challenge }: Props): number | null | string => {
```

Comment:
Hey, let's hash this typing out: this function shouldn't return either a number or a string, it should be either or.

Null is fine because its representing a case where the time can't be computed (because there are no intersections). Let's make sure its returning either `null | string` or `null | number`